packages/react/src/recipes/seo/Article/Article.module.css

```diff
@@ -28,13 +15,133 @@
   padding: 0;
 }
 
-.heroImageArea {
-  overflow: hidden;
+.hero {
+  --parallax-start-spacing: 74px;
+
+  background: black;
+  position: relative;
+  height: calc(70vh - var(--parallax-start-spacing));
+}
+
+.parallax {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 70vh;
```

Comment:
Nit.. 70vh is referenced a few times here. Worth making this a locally scoped token too like `parallax-start-spacing`?

Reply:
Fine by me — It was a CSS var a while ago (when it was exposed as a control in Storybook) and I removed it when I removed those controls.

Happy to split it out though, agree that it makes it a bit clearer 👍

Reply:
b790ac37db8404329ed24e6d738705138a3278af